external/storm-kafka-client/src/main/java/org/apache/storm/kafka/trident/TridentKafkaState.java

```diff
@@ -75,20 +75,29 @@ public void prepare(Properties options) {
     public void updateState(List<TridentTuple> tuples, TridentCollector collector) {
         String topic = null;
         try {
-            List<Future<RecordMetadata>> futures = new ArrayList<>(tuples.size());
+            long currentTime=System.currentTimeMillis();
```

Comment:
1. `startTime` would be better to represent its intention well
2. space between `e` and `=` and `=` and `S`